tsconfig.json

```diff
@@ -0,0 +1,21 @@
+{
+    "compilerOptions": {
+        "target": "es6",
+        "module": "commonjs",
+        "rootDir": "./",
```

Comment:
Would we be able to avoid a bit of extra importing if we set the rootDir as `./lib`?

Reply:
Unfortunately considering the rootDir as `./lib` would ignore `index.ts`. This is not the desired behaviour as of now, but we could move the `index.ts` in the `lib` folder.